book/10-git-internals/sections/environment.asc

```diff
@@ -220,8 +220,8 @@ Note that this isn't the easiest way to customize how `ssh` is invoked; it won't
 It's probably easier just to use the `~/.ssh/config` file for that.
 
 *`GIT_ASKPASS`* is an override for the `core.askpass` configuration value.
-This is the program invoked whenever Git needs to ask the user for credentials, which can expect a text prompt as a command-line argument, and should return the answer on `stdout`.
-(See <<ch07-git-tools#_credential_caching>> for more on this subsystem.)
+This is the program invoked whenever Git needs to ask the user for credentials, which can expect a text prompt as a command-line argument, and should return the answer on `stdout`
+(see <<ch07-git-tools#_credential_caching>> for more on this subsystem).
```

Comment:
Please, keep the following convention: "one sentence - one line". Therefore, either concatenate both lines to a single one or remove parentheses and leave the second line as an individual sentence.

Reply:
Okay, I'll merge it in a single line. @Morganov I've a question, I'm agree with all of your suggestions. But how do I incorporate these suggestions into PR? Means should I use Github `Commit suggestions` feature? or I've to make these changes locally, then force push it; so you don't have to rebase the PR while merging.

Reply:
In this particular case, I recommend you do the changes locally, amend the commit, and do the force push.
Here, I've used `commit suggestions` to show what exactly should be changed only.
IMHO, nobody wants to see the changes, that have been reverted in the same PR (unless it's necessary 😄).

So again, in the matter to keep git history in a cleaner state, I recommend you update the current commit.

If you don't know how to do this, it's OK, go ahead to apply suggestions - all commits in your PR could be squashed automatically on merge.